### DISCUSSION TITLE
Element overloads

### DISCUSSION THREAD

**Discussion by tobyck** [Vyxal/Vyxal#1337] [Polls]
We have [quite a few elements](https://chat.stackexchange.com/transcript/message/61629038#61629038) with overload space, so in this discussion we'll decide what new features get added to these elements or what gets migrated from unused digraphs. 

Overload space that has been filled since the beginning of this discussion:
- `Ṡ` (str and num)

Potential overload space:
- `⟇` (str)
- `ṁ` (num)
- `ẏ` and `ż` (str)

(The poll for `Ṡ` is still here because it can't be removed and there's currently nothing to fill the overload space with)

**Comment by Steffan153** [comment]
I'm thinking remove trailing zeros isn't really useful, so maybe leave the `num` overload for something else.

**Reply by tobyck** [reply]
Ok, any ideas for the `num` overload? I'll add them at the same time.
